### PR TITLE
826 Error in Contributions GET for Talent and Capabilities

### LINKED ISSUE #826: [BUG] Error in Contributions BB when getting Talents and Capabilities
**Describe the bug**
Contributions BB is returning HTTP 500 errors for different endpoints. This is different from the POST errors mentioned in #791 and is currently present in the development branch.

**To Reproduce**
1. POST a Contribution (status is Submitted) containing a Talent and a Capability
2. GET a Capability using:
```
curl --location --request GET 'http://localhost:5000/contributions/<contribution_id>/capabilities/<capability_id>' \
--header 'ROKWIRE-API-KEY: <ROKWIRE-API-KEY>' \
```
3. GET a Talent using:
```
curl --location --request GET 'http://localhost:5000/contributions/<contribution_id>/talents/<talent_id>' \
--header 'ROKWIRE-API-KEY: <ROKWIRE-API-KEY>' \
```
4. See HTTP 500 error in the terminal.

**Expected behavior**
It should return 401 as these resources are not accessible using the API Key (since the Contribution is not published). 

**Additional context**
 Please verify this. I think the issue is in the logic in the following block. 
```
if status == "Published":
            return dataset, status_code
else:
    if (is_login):
        # check if the user is in contributionAdmin group
        if (is_admin):
            return dataset, status_code
        else:
            status_code = '401'
            return None, status_code
```
in mongoutils.py. In this case, is_login is false and status is "Submitted". Hence this method is returning nothing. 
Unrelated: You can remove the parentheses from those statements.

### PR DESCRIPTION
**Pull request by minump**
## Description
This PR makes changes in mongoutils so as to output Authorization error (401) on GET on unpublished contributions.

**Fixes #826**
Screen Shot 2022-03-31 at 2 30 55 PM

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Review by ywkim312 — DISMISSED** [review]

**Comment by ywkim312** [comment]
please fix merge conflict

**Comment by sandeep-ps** [comment]
Hi, @minump, Please remove the PR from the GitHub Project as this is already linked to the issue. Thanks.

**Comment by minump** [comment]
> Hi, @minump, Please remove the PR from the GitHub Project as this is already linked to the issue. Thanks.

done

File: CHANGELOG.md
Comment (sandeep-ps) [inline CHANGELOG.md:27]: Please move this entry to the Unreleased section. Thanks.

**Comment by sandeep-ps** [comment]
Tested this PR with all endpoints and it's working. Thanks.

File: CHANGELOG.md
Comment (minump) [reply CHANGELOG.md]: updated

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
@ywkim312, Do you have any other feedback on this PR? My brief tests look fine, but as the code owner, I will need your review before merging. Thanks.

**Review by ywkim312 — APPROVED** [review]

**Review by ywkim312 — APPROVED** [review]

**Comment by sandeep-ps** [comment]
@minump, Could you please resolve the merge conflicts? We can merge this PR after that. Thanks.

**Comment by minump** [comment]
> @minump, Could you please resolve the merge conflicts? We can merge this PR after that. Thanks.

resolved merge conflicts

**Pull request merged**
